### ISSUE TITLE
Typescript in .vue files

### ISSUE THREAD

**Issue by ghost** [tobiasalthoff/vscode-atom-material-theme#12]
Hi, I'm using Atom Material Theme with .vue files. Lately I started using Typescript and syntax highlighting doesn't look right to me. See attached screenshots

![without-ts-lang](https://user-images.githubusercontent.com/57101430/87666236-8a5b8000-c768-11ea-8ea4-ffbe555d9213.png)
![with-ts-lang](https://user-images.githubusercontent.com/57101430/87666239-8b8cad00-c768-11ea-91f6-6f3675d94b2c.png)

`components` and `props` look like regular strings with `lang="ts"`

**Comment by tobiasalthoff** [comment]
Hey @phillipcodes,

yeah, that looks wrong :-) Could you please give me an example snippet?

**Comment by ghost** [comment]
Sure!

Agenda.vue
```
<template>
    <table>
        <thead>
            <tr>
                <th v-for="location in locations">
                    {{ location.title }}
                </th>
            </tr>
        </thead>
    </table>
</template>

<script lang="ts">
import { defineComponent, ref, computed } from 'vue'
import { Event, Location } from '../interfaces'
import EventColumn from './EventColumn.vue'

export default defineComponent({
    components:  {
        EventColumn,
    },
    props: {
        events: {
            type: Array as () => Event[],
            default: [],
        }
    },
    setup(props: { events: Event[] }) {
        const getLocations = (): Location[] => {
            const notUndefined = <T>(x: T | undefined): x is T => {
                return x !== undefined;
            }
            const allLocations = props.events
                .map(event => event.location)
                .filter(notUndefined)

            let locationIds = new Set<string>()
            allLocations.forEach(location => {
                locationIds.add(location.id)
            })

            return [...locationIds]
                .map(
                    locationId => allLocations.find(
                        location => location.id === locationId
                    )
                )
                .filter(notUndefined)
        }
        const locations = ref<Location[]>(getLocations())

        return {
            locations,
        }
    },
})
</script>
```

**Comment by tobiasalthoff** [comment]
Fixed with version v1.10.3 → update should be available now.
